chassis/core/src/main/java/com/griddynamics/jagger/invoker/v2/SpringBasedHttpClient.java

```diff
@@ -103,6 +107,13 @@ public JHttpResponse execute(JHttpEndpoint endpoint, JHttpQuery query) {
         return mapToJHttpResponse(responseEntity);
     }
 
+    public JHttpResponse execute(JHttpEndpoint endpoint) {
+        URI endpointURI = endpoint.getURI();
+        RequestEntity requestEntity = mapToRequestEntity(endpointURI);
+        ResponseEntity responseEntity = restTemplate.exchange(endpointURI, HttpMethod.GET, requestEntity, byte[].class);
+        return mapToJHttpResponse(responseEntity);
```

Comment:
It's not duplicated. 
It's better to handle cases with query and without it in different methods.